packages/docs/docs/docs/advanced.md

```diff
@@ -138,6 +138,13 @@ data = encodeCall(
 
 await exampleToken.sendTransaction( {data, from: owner} );
 ```
+## Safety checks
+
+The ZeppelinOS CLI performs a series of safety checks in some of its commands with the purpose of strengthening the security of your application. For example, the `zos add` command can detect if a contract has a `constructor`, or contains usages of `selfdestruct` or `delegatecall`. Below is a list of checks made, along with the reasoning behind why they are considered to be security risks:
+
+* constructor check: Proxied contracts should use initializer functions instead of constructors. For more info, see "Initializers vs. constructors" in this page.
+* selfdestruct check: Solidity's `selfdestruct` keyword ends the execution of a contract, destroys it, and sends all of its funds to a specified account. This is a very delicate action to take, and can expose a contract to vulnerabilities such as the [second Parity Wallet hack](https://blog.zeppelinos.org/parity-wallet-hack-reloaded/). It should be used with extreme care.
+* delegatecall check: The `delegatecall` keyword fully exposes a contract's state to a 3rd party contract. Such a contract (to which the delegatecall is made to) has complete control on the calling contract. It should only be used if you really know what you're doing.
```

Comment:
this sentence is hard to read: (to which the delegatecall is made to). Why not just removing it?